config/project_aanvraag.php

```diff
@@ -8,4 +8,5 @@
         'test' => env('PROJECT_AANVRAAG_BASE_URI_TEST', 'http://localhost/'),
         'live' => env('PROJECT_AANVRAAG_BASE_URI_LIVE', 'http://localhost/'),
     ],
+    'connect_timeout' => env('PROJECT_AANVRAAG_CONNECT_TIMEOUT', 10.0),
```

Comment:
It is the more general timeout we set in the config, there is also a connect timeout but that is not the one we set. So I would name it `PROJECT_AANVRAAG_TIMEOUT`

Reply:
[Rename timeout](https://github.com/cultuurnet/publiq-platform/pull/906/commits/4774d963c5ab14c8a1e3f74fbe6fe8b8323e54ed)